maat-scheduled-tasks/src/main/java/uk/gov/justice/laa/maat/scheduled/tasks/repository/TrialDataStatusRepositoryImpl.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.laa.maat.scheduled.tasks.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+@RequiredArgsConstructor
+public class TrialDataStatusRepositoryImpl implements TrialDataStatusRepository {
```

Comment:
Curious as to why we need this repository. It could be that the task description isn't correct, but we should be querying the `XHIBIT_TRIAL_DATA` table, so you should be able to extend the existing _XhibitTrialDataRepository_ with the new method.